test/index.js

```diff
@@ -58,6 +59,8 @@ describe('DDWAF', () => {
       'key_attack',
       'server.request.body'
     ]))
+
+    waf.dispose()
```

Comment:
why add it here but not on the test above ?
technically we shouldn't need them because dispose gets called on GC, but if you prefere to manually call it, then call it everywhere no ?